collector/slave_status.go

```diff
@@ -61,9 +76,17 @@ func ScrapeSlaveStatus(db *sql.DB, ch chan<- prometheus.Metric) error {
 		for i, col := range slaveCols {
 			if value, ok := parseStatus(*scanArgs[i].(*sql.RawBytes)); ok { // Silently skip unparsable values.
 				ch <- prometheus.MustNewConstMetric(
-					newDesc(slaveStatus, strings.ToLower(col), "Generic metric from SHOW SLAVE STATUS."),
+					prometheus.NewDesc(
```

Comment:
I guess it's an unnecessary simplification.  Nevermind, maybe I'll just pull it from the code.

Either way, please use the collector variable `namespace` instead of the string literal `"mysql"` here.